src/System.Diagnostics.Debug/src/System/Diagnostics/Debug.Unix.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.InteropServices;
+
+using Microsoft.Win32.SafeHandles;
+
+namespace System.Diagnostics
+{
+    public static partial class Debug
+    {
+        // Internal so that the tests can override this.
+        internal static IDebugLogger s_logger = new UnixDebugLogger();
```

Comment:
The tests need to swap it out at runtime :-(.  I will update the comment to make it more clear.